libs/environment/src/hooks/use-environment.ts

```diff
@@ -159,7 +163,20 @@ export const useInitializeEnv = () => {
 const fetchConfig = async (url?: string) => {
   if (!url) return [];
   const res = await fetch(url);
-  const cfg = await res.json();
+  let cfg: Record<string, unknown>;
+  if (url.match(/github(.+)\.toml$/)) {
+    const content = await res.text();
+    const parsed = tomlParse(content);
```

Comment:
what if this is not valid toml or res.text fails, probably need some kind of catch

Reply:
This is wrapped with try catch in the main body of the initialize function. If it fails initialization will continue, and the app will attempt to load via VEGA_URL or what is in localstorage

Reply:
Noice